README.md

```diff
@@ -48,33 +48,39 @@ Install each of these and ensure they are included in your PATH. Then install th
 
 The ["Getting Started" guide](Getting_Started_with_eva_sub_cli.md) serves as an introduction for users of the eva-sub-cli tool. It includes instructions on how to prepare your data and metadata, ensuring that users are equipped with the necessary information to successfully submit variant data. This guide is essential for new users, offering practical advice and tips for a smooth onboarding experience with the eva-sub-cli tool.
 
-## eva-sub-cli tool: Options and parameters guide
+## Options and parameters guide
 
-The eva-sub-cli tool provides several options/parameters that you can use to tailor its functionality to your needs. Understanding these parameters is crucial for configuring the tool correctly. Below is an overview of the key parameters and options:
+The eva-sub-cli tool provides several options and parameters that you can use to tailor its functionality to your needs.
+You can view all the available parameters with the command `eva-sub-cli.py -h` and view detailed explanations for the
+input file requirements in the ["Getting Started" guide](Getting_Started_with_eva_sub_cli.md).
+Below is an overview of the key parameters.
 
-| OPTIONS/PARAMETERS         | DESCRIPTION                                                                                                                                                                                                        |
```

Comment:
We can discuss if people think we should keep this table here as well as in the `help` command, if we do we just need to remember to keep it in sync.

I've tried to make the documentation here more of a "quick reference" and left the more detailed explanation and troubleshooting guide to the "Getting started" document. Let me know what you think about this distinction.